### PR TITLE
 feat(Tabs2): Tabs with improved interface

### PR DESCRIPTION
**Pull request by laraharrow**
- [ ] ♿️ Accessibility addressed
- [ ] 🌏 Localization & Internationalization addressed
- [ ] 🖼 Image Snapshot coverage
- [ ] 📚 Documentation updated
- [ ] 🧳 Bundle size impact addressed
- [ ] 🏁 Performance impacts addressed
- [ ] 👾 Browsers tested
  - [ ] Chrome
  - [ ] Firefox
  - [ ] Safari
  - [ ] IE11

### CODE REVIEW

File: packages/components/src/Tabs2/Tabs2.test.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  ).toBeInTheDocument()
+  expect(screen.queryByText('Are kinda smelly')).not.toBeInTheDocument()
+  fireEvent.click(screen.getByText('Fish'))
+  expect(
+    screen.queryByText("Here's awesome story about cats")
+  ).not.toBeInTheDocument()
+  expect(screen.queryByText('Are kinda smelly')).toBeInTheDocument()
+})
+
+test('clicking on disable tab does not change panel', () => {
```
Comment (lukelooker) [inline packages/components/src/Tabs2/Tabs2.test.tsx:64]:
```suggestion
test('clicking on disabled tab does not change panel', () => {
```

File: packages/components/src/Tabs2/Tabs2.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+import { Tab, TabList, TabPanels } from '../Tabs'
+
+type Tab2Props = {
+  children: ReactNode
+  disabled?: boolean
+  id?: string
+  label: string
+}
+
+export const Tab2: FC<Tab2Props> = ({ children }) => <>{children}</>
```
Comment (lukelooker) [inline packages/components/src/Tabs2/Tabs2.tsx:44]: Represents a separate component so it should be in a separate file.

**Review by lukelooker — CHANGES_REQUESTED** [review]
Noted a few items inline as well as:

- [ ] Active tab doesn't show the "key" border at the bottom.
- [ ] Screenshot tests indicate that screenshots are out of date.

File: packages/components/src/Tabs2/Tabs2.test.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  expect(
+    screen.queryByText("Here's awesome story about cats")
+  ).toBeInTheDocument()
+  expect(screen.queryByText('not available')).not.toBeInTheDocument()
+  fireEvent.click(screen.getByText('Human'))
+  expect(
+    screen.queryByText("Here's awesome story about cats")
+  ).toBeInTheDocument()
+  expect(screen.queryByText('not available')).not.toBeInTheDocument()
+})
```
Comment (lukelooker) [inline packages/components/src/Tabs2/Tabs2.test.tsx:76]: It'd be good to add a "controlled" example of `Tabs2` to make sure we have coverage of that use case within the test suite.

**Comment by laraharrow** [comment]
snapshot issue

**Pull request closed without merging**